nc-news-react/src/App.jsx

```diff
@@ -17,6 +18,7 @@ function App() {
     <Route path='/articles' element={<Articles />} />
     <Route path='/articles/:article_id' element={<SingleArticle />} />
     <Route path='/post' element={<Post />} />
+    <Route path='/articles/:article_id/comments' element={<Comments />} /> 
```

Comment:
Rather than a path, it would be nicer to have your comments component rendered within your SingleArticle one. This way, your comments list will show under the article itself and is much nicer for user experience